api/src/main/java/ai/djl/training/listener/MemoryTrainingListener.java

```diff
@@ -51,7 +51,9 @@
     public MemoryTrainingListener() {}
 
     /**
-     * Constructs a {@link MemoryTrainingListener} that outputs data in the given directory.
+     * Constructs a {@link MemoryTrainingListener} that outputs data in the given directory. If an
+     * output directory is provided, the file "$outputDir/memory.log" will be created after training
```

Comment:
We usually leave the first sentence as just a summary and move the rest to a new paragraph. Also, can you provide a brief description about the format of memory.log?